evap/staff/views.py

```diff
@@ -139,105 +144,152 @@ def __init__(self):
     return render(request, "staff_semester_view.html", template_data)
 
 
+class EvaluationOperation:
+    email_template_name = None
+    confirmation_message = None
```

Comment:
this one is unused.

speaking of which, i actually think it's a good idea, how about making the first three methods *actual* properties? just like `email_template_name`. the first one would be a list of strings, then a string and a lazy string. i also thought about making `apply` a lambda and all the `Operations` tuples, but i'm not sure that makes it better. if you want you could try that as well.